logging/src/test/java/org/jboss/as/logging/LoggingSubsystemTestCase.java

```diff
@@ -266,6 +266,36 @@ public void testFailedTransformersWildFly800() throws Exception {
                                 new FailedOperationTransformationConfig.NewAttributesConfig(SizeRotatingHandlerResourceDefinition.SUFFIX)));
     }
 
+    @Test
+    public void testTransformersEAP700() throws Exception {
+        testEap7Transformer(ModelTestControllerVersion.EAP_7_0_0, ModelVersion.create(3, 0, 0), readResource("/logging_3_0.xml"),
+                // In WildFly Core the default formatter changed from %E (extended exceptions) to %e. When the legacy
+                // model is compared to the new model and the formatter is not defined or a named-formatter is used the
+                // check fails due to the formatter difference. This is not ideal, but will work as the difference isn't
+                // important.
+                new AttributeValueChangerModelFixer(AbstractHandlerDefinition.FORMATTER, "%E", "%e")
```

Comment:
@bstansberry This was more of my testing if EAP7 legacy controller works.
This is copy-paste from testTransformersWildFly800() method.
Removed this cruft.